nima/webserver/webserver/src/main/java/io/helidon/nima/webserver/ListenerConfiguration.java

```diff
@@ -382,6 +395,18 @@ public Builder writeBufferSize(int writeBufferSize) {
             return this;
         }
 
+        /**
+         * Grace period to allow running tasks to complete before listener's shutdown.
+         * Default is {@code 500} milliseconds.
+         *
+         * @param gracePeriodMillis grace period
+         * @return updated builder
+         */
+        public Builder gracePeriodMillis(long gracePeriodMillis) {
```

Comment:
We use `Duration` for configurations such as this, so the method should be `gracePeriod\(Duration duration)`,

Reply:
Changes pushed.